nomenclature/__init__.py

```diff
@@ -69,6 +73,29 @@ def _copy_dict(dct, description):
                     _key_ccs = f'{_key}|{sub}'
                     _description_ccs = f'{_description} {desc}'
                     variables[_key_ccs] = _copy_dict(value, _description_ccs)
+
+    # if the key contains the tag, loop over all industry types to add mapping
```

Comment:
this piece of code is repetitive and could be refactored like:
```
key_types = [
    ('<Fuel>', fuel_types),
    ('<Industry>', industry_types),
    ('<Product>', product_types)
]
for k, types in key_types:
    if k in key:
        for f, attr in types():
            ...
```

Reply:
Thanks, I got it.